### PR TITLE
[Reskin-570]Add the filters for project page

### PR DESCRIPTION
**Pull request by alejandrocabriales**
## Ticket
https://trello.com/c/VIIpc7Ew/570-tpd-21-powerhouse-project-details-page

## What solved
- [X] Add the filters with the new styles

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have performed a self-review of my own chromatic changes
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have checked my code and corrected any misspellings
- [x] I have removed any unnecessary console messages
- [x] I have removed any commented code
- [x] I have checked that there are no buggy stories in Storybook

### CODE REVIEW

File: src/views/Home/components/MilestoneCard/MilestoneCard.tsx [hunk 1]
```diff
@@ -28,16 +26,12 @@ interface ElementWithProgress {
...
-  status: Maybe<DeliverableSetStatus>;
-}
-
 interface ElementWithTotal {
   total?: number;
 }
 
 const MilestoneCard: FC<MilestoneCardProps> = ({ slug, milestoneData }) => {
-  const { statusLabel } = useMilestoneCard(milestoneData.scope?.status);
+  // const { statusLabel } = useMilestoneCard(milestoneData.scope?.status);
```
Comment (YasielCabrera) [inline src/views/Home/components/MilestoneCard/MilestoneCard.tsx:34]: could you remove this commented code?

File: src/views/ActorProjects/useActorProjectsContainer.tsx [hunk 2]
```diff
@@ -89,6 +133,9 @@ const useActorProjectsContainer = (projectsData: ProjectsAndSupportedProjects, a
     filteredProjects,
     filteredSupporterProjects,
     pager,
+    canReset,
+    filters,
+    // handleResetFilters,
```
Comment (YasielCabrera) [inline src/views/ActorProjects/useActorProjectsContainer.tsx:138]: could you remove this commented code?

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/views/Home/components/MilestoneCard/MilestoneCard.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/views/Home/components/MilestoneCard/MilestoneCard.tsx]: done

File: src/views/ActorProjects/useActorProjectsContainer.tsx (hunk 2 above)
Comment (alejandrocabriales) [reply src/views/ActorProjects/useActorProjectsContainer.tsx]: done

**Review by YasielCabrera — APPROVED** [review]
lgtm

**Pull request merged**